### DISCUSSION TITLE
Authentication bug

### DISCUSSION THREAD

**Discussion by nextdev1111** [supabase/supabase#7553] [Questions] [answered]
user trys to login with email and password. Then we created a public profile for him using triggers. But what if then he trys to login with Google with same address. Will supabase create a new user or update the existing user.

**Comment by burggraf** [comment, marked as answer]
If the email address is the same, then it will update the existing user, as email address is used as a key.  This may change, however, or be configurable, at least, in a future release.  But in my apps, for now at least, you can login with email then log in with the same account using Google (or other Oauth providers) and vice-versa (log in with Oauth, then send a "forgot password" link to be able to later log in via email address.)

**Reply by nextdev1111** [reply]
Thanks
